js/controller.js

```diff
@@ -491,6 +491,20 @@ module.exports = function(OO, _, $, W) {
       this.renderSkin({cast: {connected:false, device: ""}});
     },
 
+    airPlayListener: function(event) {
+      switch (event.availability) {
+        case 'available':
+          this.state.isAirPlayAvailable = true;
+          break;
+        case 'not-available':
+          this.state.isAirPlayAvailable = false;
+          break;
+        default:
+          break;
+      }
```

Comment:
```suggestion
      this.state.isAirPlayAvailable = event.availability === 'available';
```
Untill third option of `event.availability` flag appears, let's avoid undetermined state of 502-503 lines.

Reply:
as I remember we'll have to describe the default behavior after new eslint rules are applied